ices/81199.rs

```diff
@@ -0,0 +1,18 @@
+#[repr(C)]
```

Comment:
Could you use this instead? https://github.com/rust-lang/rust/issues/81199#issuecomment-763194019

Reply:
Resolved by 4a45148